website/docs/reference/resource-configs/snowflake-configs.md

```diff
@@ -9,6 +9,399 @@ To-do:
 - use the reference doc structure for this article / split into separate articles
 --->
 
+## Iceberg table format <Lifecycle status="beta"/>
+
+The dbt-snowflake adapter supports the Iceberg table format. It is available for three of the Snowflake materializations: 
+
+- [Table](/docs/build/materializations#table)
+- [Incremental](/docs/build/materializations#incremental)
+- [Dynamic](#dynamic-tables) 
+
+For now, to create Iceberg tables, you must implement a [behavior flag](/reference/global-configs/behavior-changes) due to performance impact related to using Iceberg tables. Snowflake does not support `is_iceberg` on the `Show Objects` query, which dbt depends on for metadata.
+
+To use Iceberg, set the `enable_iceberg_materializations` flag to `True` in your dbt_project.yml:
+
+<File name='dbt_project.yml'>
+
+```yaml
+
+flags:
+  enable_iceberg_materializations: True
+
+```
+
+</File>
+
+
+The following configurations are supported: 
+
+| Field | Type | Required | Description | Sample input |
+|-------|------|----------|-------------|-------|
+|Table Format | String | Yes | Configures the object table format. | `iceberg` |
+|External volume | String | Yes | Specifies the identifier (name) of the external volume where the Iceberg table stores its metadata files and data in Parquet format. If you don’t specify this parameter, the Iceberg table defaults to the external volume for the schema, database, or account. The schema takes precedence over the database, and the database takes precedence over the account.| `my_s3_bucket` |
+|Base location Subpath |String | No | Defines the directory path for Snowflake to write the table data and metadata files. If you change this after creating the table, it will write a new table but leave the existing one. | `jaffle_marketing_folder` |
+
+### Example configuration
+
+To configure an Iceberg table materialization in dbt, refer to the example configuration:
+
+<File name='models/<modelname>.sql'>
+
+```sql
+
+{{
+  config(
+    materialized = "table",
+    table_format="iceberg",
+    external_volume="s3_iceberg_snow",
+  )
+}}
+
+select * from {{ ref('raw_orders') }}
+
+```
+
+</File>
+
+### Base location 
+
+Snowflake's `CREATE ICEBERG TABLE` DDEL requires that a `base_location` be provided. dbt defines this parameter on the user's behalf to streamline usage and enforce basic isolation of table data within the `EXTERNAL VOLUME`. The default behavior in dbt is to provide a `base_location` string that follows the below convention:
+
+`_dbt/{SCHEMA_NAME}/{MODEL_NAME} `
```

Comment:
1. "DDL" not "DDEL"
2. in-lined the `base_location` string because it's not substantial to be it's own line
```suggestion
Snowflake's `CREATE ICEBERG TABLE` DDL requires that a `base_location` be provided. dbt defines this parameter on the user's behalf to streamline usage and enforce basic isolation of table data within the `EXTERNAL VOLUME`. The default behavior in dbt is to provide a `base_location` string of the form: `_dbt/{SCHEMA_NAME}/{MODEL_NAME}`
```